app/components/tables/utilities/page-size-input.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@ember/component';
+import { A } from '@ember/array';
+
+export default Component.extend({
```

Comment:
Lets start using ES6 classes + decorators for all new resources. That's the way ember is moving towards.